pom.xml

```diff
@@ -854,6 +855,11 @@
                 <artifactId>javax.servlet-api</artifactId>
                 <version>${servlet.version}</version>
             </dependency>
+            <dependency>
```

Comment:
Do we still need glassfish?

Reply:
I pulled it up to avoid having multiple `el` versions in the classpath and override the two versions we previously had. Actually, we should get rid of the `javax.*` namespace in the long run and migrate to `jakarta.*` anyway. In this process, we can switch to the Eclipse foundation spec / api jars which are good to use in an ASF project.